### DISCUSSION TITLE
Themes

### DISCUSSION THREAD

**Discussion by protean-onion** [sagb/alttab#175] [Q&A]
Hello,
I love this app, though I'd like to change how it appears. Are there any other themes that can be installed?
Thanks.

**Comment by sagb** [comment]
There aren't any specific themes as such, but there are many individual options that control the appearance.  
Look for the following options in `man alttab`: `-t`, `-i`, `-bg`, `-fg`, `-frame`, `-inact`, `-font`, `-vertical`, `-b`.

**Comment by sagb** [comment]
To be honest, it looks like you had a bunch of scripts generating nice outputs along with the "ls" command...
I'd appreciate a PR for an improved screenshots.md, though.
